### DISCUSSION TITLE
Armor Stand poses!

### DISCUSSION THREAD

**Discussion by justmyrame** [PurpurMC/Purpur#479] [__Dismissed__] [answered]
Hi!
I have been using Purpur for a long time, and this is the best fork I have ever used. I am now preparing a server for my friends and while looking for ideas on the Internet, I found out that in the Bedrock Edition, which is slightly different from the Java version, you can change Armor Stand pose.

> ![Pose Gif](https://qph.fs.quoracdn.net/main-qimg-89175c08b78ba0942061df671601f7a6)
> *GIF source: Quora*

Immediately I started looking for a way to add it to the server. Unfortunately, I did not find a plugin that would add it.
I'd like to suggest adding this feature to Purpur. In my opinion, this will be a useful tool for multiple SMP servers.

**Comment by emielderckx** [comment]
This is more a plugin thing. https://www.spigotmc.org/resources/edit-armor-stands.5445/
or use the vanillatweaks datapack

**Comment by BillyGalbreath** [comment, marked as answer]
You already can with vanilla ^_^

```
/summon armor_stand ~ ~ ~ {ShowArms:1b,Rotation:[28f],Pose:{Head:[18f,13f,7f],LeftLeg:[26f,5f,1f],RightLeg:[322f,7f,5f],LeftArm:[28f,19f,326f],RightArm:[224f,69f,30f]}}
```

https://haselkern.com/Minecraft-ArmorStand/

**Reply by justmyrame** [reply]
Yes, it is possible with the command, I mean to import this functionality from bedrock, change the Armor Stand pose to one of the default ones with Shift + Right click

**Reply by MrButtersDEV** [reply]
> Yes, it is possible with the command, I mean to import this functionality from bedrock, change the Armor Stand pose to one of the default ones with Shift + Right click
> 
> 

I don't think this is something that would be added to a java server fork but its definitely something that can be done with a plugin! Feel free to dm me on discord and I could maybe make something. BUTTERFIELD8#3907
